tap_exacttarget/endpoints/campaigns.py

```diff
@@ -36,6 +36,7 @@ class CampaignDataAccessObject(DataAccessObject):
 
     TABLE = 'campaign'
     KEY_PROPERTIES = ['id']
+    REPLICATION_METHOD = 'FULL_TABLE'
```

Comment:
@harshpatel4crest 'REPLICATION_KEYS' variable is not set for the 'Campaigns' stream. Does this mean that this stream does not have any replication keys?

Reply:
Yes, it is FULL_TABLE